### PR TITLE
Job now pulls image based on hash instead of :latest tag

### PR DESCRIPTION
**Pull request by csheremeta**
This needs to merge AFTER https://github.com/openshift/must-gather-operator/pull/66

### CODE REVIEW

File: pkg/controller/mustgather/mustgather_controller.go [hunk 1]
```diff
@@ -136,7 +137,11 @@ func initializeTemplate() (*template.Template, error) {
 		log.Error(err, "Error reading job template file", "filename", templateFileName)
 		return &template.Template{}, err
 	}
-	jobTemplate, err := template.New("MustGatherJob").Parse(string(text))
+	// Inject the operator image URI from the pod's env variables
+	operator_image := os.Getenv("OPERATOR_IMAGE")
```
Comment (dustman9000) [inline pkg/controller/mustgather/mustgather_controller.go:141]: Can we throw an error if OPERATOR_IMAGE is not found or is nil?

**Comment by 2uasimojo** [comment]
looks good

**Comment by bng0y** [comment]
Could we document what changes for local development? 

- do we need to add an env variable to `https://github.com/openshift/must-gather-operator/blob/master/deploy/99_must-gather-operator.Deployment.yaml#L36`
- or do need to export a new variable in `https://github.com/openshift/must-gather-operator/blob/master/README.md#local-development`

**Comment by csheremeta** [comment]
great catch, @bng0y :) README has been updated

**Pull request merged**